components/kubearchive/production/kflux-ocp-p01/kustomization.yaml

```diff
@@ -0,0 +1,176 @@
+      spec:
+        dataFrom:
+          - extract:
+              key: integrations-output/external-resources/appsres09ue1/kflux-ocp-p01/kube-archive-staging-rds
+  # These patches add an annotation so an OpenShift service
+  # creates the TLS secrets instead of Cert Manager
+  - patch: |-
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: kubearchive-api-server
+        namespace: kubearchive
+        annotations:
+          service.beta.openshift.io/serving-cert-secret-name: kubearchive-api-server-tls
+  - patch: |-
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: kubearchive-operator-webhooks
+        namespace: kubearchive
+        annotations:
+          service.beta.openshift.io/serving-cert-secret-name: kubearchive-operator-tls
+  - patch: |-
+      apiVersion: admissionregistration.k8s.io/v1
+      kind: MutatingWebhookConfiguration
+      metadata:
+        name: kubearchive-mutating-webhook-configuration
+        annotations:
+          service.beta.openshift.io/inject-cabundle: "true"
+  - patch: |-
+      apiVersion: admissionregistration.k8s.io/v1
+      kind: ValidatingWebhookConfiguration
+      metadata:
+        name: kubearchive-validating-webhook-configuration
+        annotations:
+          service.beta.openshift.io/inject-cabundle: "true"
+  # These patches solve Kube Linter problems
+  - patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: kubearchive-api-server
+        namespace: kubearchive
+      spec:
+        template:
+          spec:
+            containers:
+              - name: kubearchive-api-server
+                env:
+                - name: KUBEARCHIVE_OTEL_MODE
+                  value: enabled
+                - name: OTEL_EXPORTER_OTLP_ENDPOINT
+                  value: http://otel-collector:4318
+                securityContext:
+                  readOnlyRootFilesystem: true
+                  runAsNonRoot: true
+  - patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: kubearchive-operator
+        namespace: kubearchive
+        annotations:
+          ignore-check.kube-linter.io/unset-memory-requirements: >
+            "This is temporary to troubleshoot an issue"
+
+      spec:
+        template:
+          spec:
+            containers:
+              - name: manager
+                env:
+                - name: KUBEARCHIVE_OTEL_MODE
+                  value: enabled
+                - name: OTEL_EXPORTER_OTLP_ENDPOINT
+                  value: http://otel-collector:4318
+                securityContext:
+                  readOnlyRootFilesystem: true
+                  runAsNonRoot: true
+                ports:
+                  - containerPort: 8081
+                resources:
+                  $patch: replace
+                  limits:
+                    cpu: 500m
+                  requests:
+                    cpu: 10m
+                    memory: 256Mi
```

Comment:
to prevent eviction of this service in production, we'd like to ensure the deployment has Guaranteed QoS (i.e. `limits==requests`). This suggestion applies also to the other KubeArchive's deployments.

Reply:
It applies for both, I checked the docs.